### DISCUSSION TITLE
How to make grid for desktop and standard for mobile

### DISCUSSION THREAD

**Discussion by YouCanflyThruMostTheSun** [nolimits4web/swiper#5745] [Q&A]
On this website: https://moonmagic.com/products/raw-crystal-ring-flow?variant=39414850617416

They are doing something pretty neat that I would like to use on my site but I am having trouble figuring out how they did it. 

They are using what I think is a grid pattern for desktop but on mobile, it changes to a standard view with navigation arrows.

**Comment by Arhell** [comment]
they rewrote the css to suit their needs
also used
https://swiperjs.com/swiper-api#param-breakpoints
